src/apiUrls.ts

```diff
@@ -0,0 +1,10 @@
+export const gatewayUrl = process.env.GATEWAY_URL;
+
+export const apiEndpoints = {
+  kiosk: "/kiosk-api",
+  scrapers: "/scrapers-kiosk-api",
+};
```

Comment:
right now you are not using it anywhere. please modify this object to hold values like `${gatewayUrl}/kiosk-api`